package.json

```diff
@@ -25,7 +25,7 @@
     "jasmine": "^2.9.0"
   },
   "dependencies": {
-    "cqm-models": "projecttacoma/cqm-models",
+    "cqm-models": "projecttacoma/cqm-models#measure_model",
```

Comment:
I'm guessing the hash means a branch name? If so, should the measure model be merged before this, and this should just point to master? If my guess was incorrect, ignore this.

Reply:
That is correct, yes